src/status/TransactionConfirmed.tsx

```diff
@@ -88,6 +88,12 @@ const TransactionConfirmed = () => {
                 <div>
                     <h2>{t("tx_confirmed")}</h2>
                     <p>{t("tx_ready_to_claim")}</p>
+                    <Show
+                        when={[SwapType.Chain, SwapType.Reverse].includes(
+                            swap().type,
+                        )}>
+                        <h3>{t("tx_in_mempool_warning")}</h3>
+                    </Show>
```

Comment:
Why should we show this warning here too? Claiming swaps should be almost instant

Reply:
True, but also doesn't matter then doesn't it? Maybe even nicer too keep the warning to avoid a visual "jump" of text when the state switches to `Claiming...`

Reply:
Ah you mean for Reverse Swaps. Good point. Useful for https://github.com/BoltzExchange/boltz-web-app/issues/346, but not when user accepts 0-conf.